src/actions/userActions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { SET_SIGNUP_FORM_INPUT, SET_SUBMIT_USERS, USER_ERRORS } from './type';
+
+export const setSignupInput = data => async (dispatch) => {
```

Comment:
Why do you have async here?

Reply:
I wanted to use promises, in case a request wasn't successful to be able to catch errors.

Reply:
async goes with await so in this case, it is irrelevant you are not using it anywhere. If the `then` fails errors will be captured in the catch.